README.md

```diff
@@ -27,11 +33,11 @@ Run Instructions for llvm-spirv
 
 llvm-spirv only accepts SPIR 1.2/2.0 or LLVM bitcode following a [SPIR-V friendly format](https://github.com/KhronosGroup/SPIRV-LLVM/blob/khronos/spirv-3.6.1/docs/SPIRVRepresentationInLLVM.rst).
 
-The [Open Source SPIR producer](https://github.com/KhronosGroup/SPIR) can be used to compile OpenCL 1.2/2.0 C source code to SPIR 1.2/2.0.
+The [Khronos OpenCL C compiler](https://github.com/KhronosGroup/SPIR) can be used to compile OpenCL 1.2/2.0 C source code to SPIR 1.2/2.0 or SPIR-V. It has three branches: spir_12, spir_20_provisional, and spirv-1.0, corresponding to SPIR 1.2, SPIR 2.0, and SPIR-V, respectively. The spirv-1.0 branch emits SPIR-V directly.
 
 To translate between SPIR 1.2/2.0 and SPIR-V:
 
-1. Follow the instructions provided by the open source SPIR producer website to use clang to compile OpenCL C source code to SPIR 1.2/2.0 binary.
+1. Follow the instructions provided by the Khronos OpenCL C compiler website to use clang to compile OpenCL C source code to SPIR 1.2/2.0 binary.
```

Comment:
You had a link back to the SPIR repo or the corresponding README from that repo in all previous cases, why not here as well? :-)